Formula/shellshare.rb

```diff
@@ -0,0 +1,16 @@
+class Shellshare < Formula
+  desc "Live Terminal Broadcast"
+  homepage "https://shellshare.net"
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
+
+  depends_on :python
```

Comment:
Use `depends_on :python if MacOS.version <= :snow_leopard` to just depend on Python 2.7.